config/locales/en.yml

```diff
@@ -31,3 +31,9 @@
 
 en:
   hello: "Hello world"
+  csv:
+    upload_success: 'Your file has been uploaded!'
+    upload_btn: 'Upload a CSV File'
```

Comment:
```suggestion
    upload_btn: 'Upload a CSV File'
    upload_success: 'Your file has been uploaded!'
```

I recommend to order YML alphabetically for 2 reasons:
- it's easier to find/check a key later
- if you use `phrase` (in most client projects), we regularly need to `pull` the translations (as they get updated by PM/Client in the phrase cloud) — pulled translations will be ordered ABC, so it means less code changes when we already have the same ordering :)